lib/wallet/test/unit/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -686,7 +686,7 @@ spec_balanceTransaction = describe "balanceTransaction" $ do
                 balance (withNoUTxO pingPong_2)
                     `shouldBe` Left
                         (ErrBalanceTxUnresolvedInputs $ NE.fromList
```

Comment:
I was going to change this to `NE.singleton` but that is only available in GHC 9 or later.